introduction.md

```diff
@@ -13,12 +13,12 @@ It's built on V8, Rust, and Tokio.
   support `fetch()`.
 - Secure by default. No file, network, or environment access unless explicitly
   enabled.
-- Supports [TypeScript](./typescript/) out of the box.
+- Supports [TypeScript](./typescript.md) out of the box.
 - Ships a single executable (`deno`).
-- Provides built-in [development tooling](./tools) like a code formatter
+- Provides built-in [development tooling](./tools.md) like a code formatter
   ([`deno fmt`](./tools/formatter.md)), a linter
-  ([`deno lint`](./tools/linter.md)), a test runner ([`deno test`](./testing)),
-  and a
+  ([`deno lint`](./tools/linter.md)), a test runner
+  ([`deno test`](./testing.md)), and a
```

Comment:
These are unrelated change, please remove

Reply:
These changes link to the correct file on GitHub, but since you requested, I will put them in another PR.